contrib/completion/bash/docker

```diff
@@ -514,14 +514,14 @@ _docker_info() {
 
 _docker_inspect() {
 	case "$prev" in
-		--format|-f)
+		--format|-f|--type)
```

Comment:
Completion should be more specific here, i.e. complete to `image` and `container` instead of accepting any input. Here's an [example](https://github.com/docker/docker/blob/master/contrib/completion/bash/docker#L223).